src/clj/clj_diff/miller.clj

```diff
@@ -177,7 +183,7 @@
   "Calculate the sequence of edits from the map of farthest reaching end
   points."
   [opts a b p delta graph]
-  (let [next-fn (partial next-edit opts a b graph delta)]
```

Comment:
re: https://github.com/droitfintech/clj-diff/pull/7#discussion_r947631849 In the future, instead of putting the `opts` at the front to fit `partial`, just create a new function that takes `[p x k]` i.e.
```clojure
(fn next-fn [p x k]
  (next-edit a b graph delta p x k opts))
```
It's both clearer what the args and intentions are and you know if you've gotten the arity wrong using `next-fn`.

Reply:
ok